AUTHORS

```diff
@@ -0,0 +1 @@
+
```

Comment:
# Names should be added to this file as
# Name or Organization <email address>
# The email address is not required for organizations.

# You can update this list using the following command:
#
# (head -n10 AUTHORS && git shortlog -se | sed -E 's/^\s+[0-9]+\t//') | tee AUTHORS
#
# Please keep the list sorted.


put the above comment in top the file & run this command -` (head -n10 AUTHORS && git shortlog -se | sed -E 's/^\s+[0-9]+\t//') | tee AUTHORS
`
this will give the Author list.